dspace/modules/api/src/main/java/org/dspace/curate/ItemsInReview.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.curate;
+
+import java.util.Date;
+import java.text.DateFormat;
+import java.util.Locale;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.DSpaceObject;
+import org.dspace.content.DCValue;
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+import org.dspace.core.Constants;
+
+import org.dspace.workflow.WorkflowItem;
+import org.datadryad.api.DryadDataPackage;
+import org.dspace.storage.rdbms.DatabaseManager;
+import org.dspace.storage.rdbms.TableRow;
+
+import org.apache.log4j.Logger;
+
+/**
+ * ItemsInReview reports on the status of items in the review workflow.
+ *
+ * The task succeeds if it was able to calculate the correct result.
+ *
+ * Input: a collection (any collection)
+ * Output: a CSV indicating simple information about the data packages that are in review
+ *
+ * @author Ryan Scherle
+ */
+@Distributive
+public class ItemsInReview extends AbstractCurationTask {
+
+    private static Logger log = Logger.getLogger(FileSimpleStats.class);
+    private static DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
+    Context context;
+
+    @Override 
+    public void init(Curator curator, String taskId) throws IOException {
+        super.init(curator, taskId);
+
+        try {
+            context = new Context();
```

Comment:
Does this context need to be completed at some point? In AbstractUnitTest, when a context is instantiated in setup(), it gets context.complete() in teardown().